### DISCUSSION TITLE
"SimpleStorage" -- insufficient funds for gas * price + value.

### DISCUSSION THREAD

**Discussion by crazywowmen** [ConsenSys-archive/truffle#5383] [General Q&A] [answered]
I was following this doc: https://trufflesuite.com/boxes/polygon/ to kickstart a simple polygon project, however i ran into problem on insufficient funds. Would love to get support on this issue.

**Comment by apdan510** [comment, marked as answer]
Hey @dongjinxujill 

Any chance you can share the repo of the code you are working on. We would love to take a look and see what is happening.

Kind regards,
Alex | Truffle Support

**Reply by crazywowmen** [reply]
hey i figure it out! to benefit people who might run into similar issues: essentially i should have test matic token for the first wallet account metamask secret phrase could recover to. so i end up adding test matic token to all my metamask accounts to make it work!

**Reply by crazywowmen** [reply]
#close
